src/app/public/modules/toast/toast-instance.ts

```diff
@@ -13,6 +13,9 @@ export class SkyToastInstance {
     return this._closed;
   }
 
+  /**
+   * An observable that indicates when the toast is closed.
```

Comment:
We have a section for `SkyToastInstance` properties in the current docs at https://developer.blackbaud.com/skyux/components/toast#skytoastinstance-properties, but the preview here doesn't generate that table. Do we need to do something to make it show up?

Reply:
Good catch. We have an issue filed for this, here: https://github.com/blackbaud/skyux-docs-tools/issues/30. I'll work on a fix for this ASAP.

Reply:
@Blackbaud-SteveBrush, given that we have an issue for this and I have the JSDocs comments in place, is there anything else to do here in this PR? Can we just merge this PR and wait for the table to be automatically generated when the `docs-tools` issue is resolved? Or do we need to hold off on merging? Or merge and then follow up later with a manual step after that `docs-tools` issue is addressed?

Reply:
I upgraded the `docs-tools` version, and that created the missing table for `SkyToastInstance`. So I think this PR should be good to go, but 2 follow-up questions:
- The current docs include both the `close()` and `closed` properties. I added the JSDocs comment for `closed`, but `docs-tools` does not create it. Is this correct? I assume so given that `closed` is private, but I just want to make sure.
- The current docs also indicate that both properties are required. I don't think this makes sense and that it's probably an error in the current docs, but I want to confirm that's correct.

Reply:
- Did you add the docs above this property? https://github.com/blackbaud/skyux-toast/blob/master/src/app/public/modules/toast/toast-instance.ts#L12.
- Yeah, they shouldn't be required.

Reply:
Looks like I had the `closed` property JSDocs comment on the wrong line (line 17, which is what this comment thread started on), but when I move it to line 12 as you indicate and serve locally, that property still doesn't show up in the docs:
![image](https://user-images.githubusercontent.com/7850105/91051768-8edc4980-e5ee-11ea-80eb-00817e21cf87.png)
![image](https://user-images.githubusercontent.com/7850105/91051801-9d2a6580-e5ee-11ea-9479-382af9fc16a2.png)
Any idea what I'm doing wrong?

Reply:
I'll take a look.

Reply:
I have a PR in docs-tools that will fix this.

Reply:
https://github.com/blackbaud/skyux-docs-tools/pull/102

Reply:
I upgraded docs-tools and this is working now.

Reply:
Thanks, Steve.